pandas/core/series.py

```diff
@@ -1891,7 +1891,7 @@ def count(self, level=None):
         2
         """
         if level is None:
-            return notna(self._values).sum()
+            return notna(self._values).sum().astype("int64")
```

Comment:
All paths for Series.count and DataFrame.count result in int64 except for this one.